cylc/flow/simulation.py

```diff
@@ -215,5 +269,5 @@ def sim_task_failed(
         sim_conf['fail cycle points'] is None  # i.e. "all"
         or point in sim_conf['fail cycle points']
     ) and (
-        try_num == 1 or not sim_conf['fail try 1 only']
+        submit_num == 0 or not sim_conf['fail try 1 only']
```

Comment:
Behaviour change?

Reply:
I shouldn't have been using `try_num`, but `submit_num`, so yes. It's the correct behaviour though.

Reply:
I'm not sure submit_num is correct here?

Reply:
Not sure. Putting it back as it was.

Reply:
Tis not how it was.